packages/frameworks/frameworks.json

```
@@ -143,6 +139,34 @@
       }
     }
   },
+  {
+    "name": "Docusaurus",
+    "slug": "docusaurus",
```

```suggestion
    "slug": "docusaurus-1",
```

Perhaps we should be specific here?

---

@styfle no, that would break existing configs